README.md

```diff
@@ -138,8 +138,9 @@ The following configuration keys are supported:
     server is on a non-standard port. See: [wrap-ssl-redirect][9].
 
   - `:xss-protection` -
-    Enable the X-XSS-Protection header that tells supporting browsers
-    to use heuristics to detect XSS attacks. See: [wrap-xss-protection][10].
+    **Deprecated.** Enable the X-XSS-Protection header. This is no
+    longer considered best practice by OWASP and should be avoided.
```

Comment:
Let's add a link around `no longer considered best practice` that links to the MDN page on it, and lets remove `by OWASP` since the OWASP page makes no mention of it now.

Reply:
Changed as suggested.